src/api/requestsApi.ts

```diff
@@ -401,3 +414,47 @@ const transformRequestFromSP = (request: any): RPARequest => {
     linkedinSearchComments: request.linkedinSearchComments,
   };
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const transformPagedRequestsFromSP = (requests: any) => {
+  const returnObject: {
+    results: PagedRequest[];
+    hasNext: boolean;
+    getNext: () => void;
+  } = {
+    results: [],
+    hasNext: requests.hasNext,
+    getNext: requests?.getNext,
+  };
+
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  requests.results.forEach((request: any) => {
```

Comment:
You know the type, so no need for an any
```suggestion
  requests.results.forEach((request: PagedRequest) => {
```